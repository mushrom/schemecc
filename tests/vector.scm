(let ((v (vector #\a #\b #\c)))
  (vector-set! v 1 10)
  (cons
    (vector-ref v 2)
    (cons
      (vector-ref v 1)
      (cons
        (vector-ref v 0)
        ()))))
