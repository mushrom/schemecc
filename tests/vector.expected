(#\c . (10 . (#\a . ())))
